Formula/e/eatmemory.rb

```diff
@@ -0,0 +1,15 @@
+class Eatmemory < Formula
+  desc "Simple program to allocate memory from the command-line"
+  homepage "https://github.com/julman99/eatmemory"
+  url "https://github.com/julman99/eatmemory/archive/refs/tags/v0.1.7.tar.gz"
+  sha256 "93a3810024df8428165322d9289cf8dadf81540a181f104c151381507fb6977e"
+  license "MIT"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/eatmemory", "-?"
```

Comment:
can you `assert_match` the response?

Also would be add an additional version check.

Reply:
Version is not printed with `-?` switch, and there is no version switch. Should I just check for exit code? I could match the current output but it would make it harder to bump version up in case the output changes. in future versions

This is the current output of `-?`:
```
Usage: eatmemory <size>
Size can be specified in megabytes or gigabytes in the following way:
#          # Bytes      example: 1024
#M         # Megabytes  example: 15M
#G         # Gigabytes  example: 2G
```

Reply:
Nevermind I will update `eatmemory` to return version on `-?` and do the proper assertion

Reply:
This is the new output for `-?` and now in the Formula there is an `assert_match` that validates the first line and version:

```
eatmemory 0.1.8 - https://github.com/julman99/eatmemory

Usage: eatmemory <size>
Size can be specified in megabytes or gigabytes in the following way:
#          # Bytes      example: 1024
#M         # Megabytes  example: 15M
#G         # Gigabytes  example: 2G
```